pkgs/development/python-modules/python-zunclient/default.nix

```
@@ -0,0 +1,93 @@
+{
+  lib,
+  buildPythonPackage,
+  docker,
+  fetchFromGitea,
+  keystoneauth1,
+  openstackdocstheme,
+  osc-lib,
+  oslo-i18n,
+  oslo-log,
+  oslo-serialization,
+  oslo-utils,
+  pbr,
+  pythonOlder,
+  setuptools,
+  sphinxHook,
+  stestr,
+  websocket-client,
+}:
+
+buildPythonPackage rec {
+  pname = "python-zunclient";
+  version = "5.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitea {
+    domain = "opendev.org";
+    owner = "openstack";
+    repo = "python-zunclient";
+    rev = version;
+    hash = "sha256-EVfrxSc/eHYZR0FGFnNAxFCiXangt8uRkAC2zpwWqcA=";
+  };
+
+  env.PBR_VERSION = version;
+
+  build-system = [
+    openstackdocstheme
+    pbr
+    setuptools
+    sphinxHook
+  ];
+
+  sphinxBuilders = [ "man" ];
+
+  # python-openstackclient is unused upstream
+  # and will cause infinite recursion in openstackclient-full package.
+  #
+  pythonRemoveDeps = [ "python-openstackclient" ];
+
+  dependencies = [
+    docker
+    keystoneauth1
+    osc-lib
+    oslo-i18n
+    oslo-log
+    oslo-serialization
```

```suggestion
    prettytable
```
https://opendev.org/openstack/python-zunclient/src/tag/5.0.0/requirements.txt

---

Thanks for your review! I've handled all your feedback :)